packages/terra-time-input/src/TimeUtil.js

```diff
@@ -1,4 +1,39 @@
 class TimeUtil {
+  /**
+   * Finds the equivalent twelve hour time and meridiem for a given hour, anteMeridiem, and postMeridiem
+   * @param {String} hour Hour to parse.
+   * @param {String} anteMeridiem String representation of the anteMeridiem.
+   * @param {String} postMeridiem String representation of the postMeridiem.
+   * @return {{hourString: string., meridiem: string}} Hour and meridiem associated to the passed in variables.
```

Comment:
nit: looks like there is an extra `.` before the `,`

Reply:
Cleaned up here. https://github.com/cerner/terra-core/pull/1027/commits/0ef896ee9d919d30bdd68359160709c20a303ed0